primer/index.bs

```diff
@@ -710,25 +736,29 @@ Token Header:
         "alg": "EC",
         "crv": "P-256",
         "x": "N6VsICiPA1ciAA82Jhv7ykkPL9B0ippUjmla8Snr4HY",
-        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s",
+        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s"
     }
 }
 ```
+
 - `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` because we
     generated the keys using eliptic curves.
 - `"typ": "dpop+jwt"`: The type of token. All DPoP Tokens should have a type of "dpop+jwt"
 - `"jwk": { "kty": "EC" ... }`: The client's public key.
 
 Token Body:
+
 ```json
 {
-    "htu": "https://auth.otherpod.example/token",
+    "htu": "https://bob.otherpod.example/private/photo_album.ttl",
     "htm": "POST",
-    "jti": "fb1264dd-fff1-4509-a7b1-0fe58d08d3e1",
-    "iat": 1603389616
+    "jti": "fb1264dd-fff1-4509-a7b1-0fe58d08d3e1"
+    "iat": 1603389616,
+    "exp": 1603393216
 }
 ```
-- `"htu": "https://auth.otherpod.example/token"`: htu limits the token for
+
+- `"htu": "https://bob.otherpod.example/private/photo_album.ttl"`: htu limits the token for
     use only on the given url.
```

Comment:
```suggestion
- `"htu": "https://auth.otherpod.example/token"`: the htu specifies the URL from which the token was issued, binding it to the issuing authorization server.
```

@elf-pavlik I think this wording should clarify things?

Reply:
This is incorrect. The `htu` property indicates the target URL used with the access token. For example, if I have an access token that I want to send to `https://example.com/container/resource.jpeg`, the DPoP proof will need to have an `htu` claim that points at `https://example.com/container/resource.jpeg`.

The access token is bound to the issuing authorization server, not the DPoP proof.

Reply:
Oops! I was lacking some context here. The URL `https://auth.otherpod.example/token` is correct, because this is where a client binds a DPoP proof to the access token.

Reply:
Yeah, so, I'm trying to find a way to clarify the wording here, such that folks don't confuse the token endpoint with the resource in the future, because both you (@acoburn) and I have mistaken that on first glance.

Reply:
I think the confusion comes from ID Token being bound not the access token. The client pushes the DPoP bound access token to the AS as UMA claim.
Once an access token gets issued from the AS it doesn't have to be DPoP bound anymore, but it's up to AS and RS how they want to use access tokens, out of scope for Solid-OIDC.
@ThisIsMissEm I'm afraid the change of working is incorrect. Only the URL should be changed since the request is limited to `https://auth.otherpod.example/token` but this has nothing to do with what kind of response is expected from there. It just happens that an access token will be provided by AS in this case but the DPoP binding is for the ID Token that was sent in the request. Solid-OIDC use of DPoP ends here and again AS and RS can use whatever access token strategy they want.